packages/server/graphql/httpGraphQLHandler.ts

```diff
@@ -64,21 +65,28 @@ const httpGraphQLBodyHandler = async (
   })
 }
 
+const contentTypeBodyParserMap = {
+  'application/json': parseBody,
+  'multipart/form-data': parseFormBody
+}
+
 const httpGraphQLHandler = uWSAsyncHandler(async (res: HttpResponse, req: HttpRequest) => {
-  const contentType = req.getHeader('content-type')
   const connectionId = req.getHeader('x-correlation-id')
   const authToken = getReqAuth(req)
   const ip = uwsGetIP(res, req)
-  if (contentType.startsWith('application/json')) {
-    const body = await parseBody(res)
-    if (!body) {
-      res.writeStatus('422').end()
-      return
-    }
-    await httpGraphQLBodyHandler(res, body, authToken, connectionId, ip)
-  } else {
+  const contentTypeHeader = req.getHeader('content-type')
+  const ct = Object.keys(contentTypeBodyParserMap).find((key) => contentTypeHeader.startsWith(key))
```

Comment:
ah i think it's just because the header can have more content to it, e.g.:

```
Content-Type: text/html; charset=UTF-8
Content-Type: multipart/form-data; boundary=something
```